libcaf_core/caf/detail/type_traits.hpp

```diff
@@ -865,7 +864,7 @@ template <class T>
 struct is_stl_tuple_type {
   template <class U>
   static auto sfinae(U*)
-    -> decltype(std::integral_constant<bool, std::tuple_size<U>::value >= 0>{});
+    -> decltype(std::integral_constant<bool, std::tuple_size<U>::value>{});
```

Comment:
This change looks very suspicious. You've probably meant to change this to `std::tuple_size_v<U> >= 0>`?